gef.py

```diff
@@ -6404,6 +6404,46 @@ def do_invoke(self, argv):
         gef_print(chunk.psprint())
         return
 
+@register_command
+class GlibcHeapChunkRangeCommand(GenericCommand):
+    """Display information on all heap chunks within memory region {from, to}.
+    First address should be valid chunk address while the last one could be arbitrary > first.
+    See https://github.com/sploitfun/lsploits/blob/master/glibc/malloc/malloc.c#L1123."""
+
+    _cmdline_ = "heap chunkrange"
+    _syntax_  = "{:s} LOCATION_FROM LOCATION_TO".format(_cmdline_)
```

Comment:
`START_LOCATION` and `END_LOCATION` would be more consistent with GDB command arguments

See https://sourceware.org/gdb/current/onlinedocs/gdb/Dump_002fRestore-Files.html